### PR TITLE
Add credit requirements

### PR DESCRIPTION
**Pull request by dsouvage**
* fix typo in course selection e.g. 1p00
* add new section for credit requirements

### CODE REVIEW

File: src/assets/cs-guide.md [hunk 1]
```diff
@@ -67,6 +71,16 @@ Students in other disciplines may obtain a minor in Applied Computing within the
...
 
+## Credit Requirements for Bachelors of Science, Computer Science
+
+Note: subject to change, check your degree audit page ( https://my.brocku.ca/BrockDB/daud_DegreeOverview.aspx ), if you're in a specialized major then this could be different for you.
+
+| Courses Numbered | 1x99 or less | 2x00 to 2x89 | 2x90 to 3x89 | 3x90 or above |
+|-------------|:------------------------------------------------:|-------------|-------------|---------------------|----------|
+| Credits Achieved| No more than 8 credits allowed| Must have at least 6 credits above | Must have at least 3 credits or above | Must have at least 3 credits or above |
+
+Use this chart in tandem with the course calender( https://brocku.ca/webcal/current/undergrad/cosc.html ) to achieve your degree requirements.
```
Comment (JoelGritter) [inline src/assets/cs-guide.md:82]: *calendar not calender

**Review by JoelGritter — CHANGES_REQUESTED** [review]
typo but otherwise 👍

File: src/assets/cs-guide.md (hunk 1 above)
Comment (firefelix) [reply src/assets/cs-guide.md]: doh, i always make that mistake. good catch.

**Review by JoelGritter — APPROVED** [review]
LGTM

**Comment by JoelGritter** [comment]
@kindeep Not sure why I don't have merging permissions, so passing it off to you.

**Comment by kindeep** [comment]
@JoelGritter Huh, that's weird. Probably some misconfig, will look into it.

Merging for now

**Pull request merged**